src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/_help.py

```diff
@@ -84,6 +199,8 @@
             For resource group:         --source-resource-id /subscriptions/{SubID}/resourceGroups/rg1
             For EventGrid topic:        --source-resource-id /subscriptions/{SubID}/resourceGroups/rg1/providers/Microsoft.EventGrid/topics/t1
             For storage account:        --source-resource-id /subscriptions/{SubID}/resourceGroups/rg1/providers/Microsoft.Storage/storageaccounts/sa1
+            For EventGrid domain:       --source-resource-id /subscriptions/{SubID}/resourceGroups/rg1/providers/Microsoft.EventGrid/domains/d1/topics/t1
```

Comment:
Minor typo: Since this is for a domain, the "/topics/t1" at the end should be removed.